jetstream/engine/token_utils.py

```diff
@@ -33,6 +33,21 @@
 # ResultToken class to store tokens ids.
 ResultTokens = Any
 
+DEFAULT_PREFILL_BUCKETS = [
+    16,
+    32,
+    64,
+    128,
+    256,
+    512,
+    1024,
```

Comment:
It wouldn't compile sequence length 32768 because of this logic: https://github.com/google/JetStream/pull/92/files/6702d2f136226b1b0417ac7cf8412540d7ef0f29#diff-38fa7646a82d9a00519f201908d44aca60d5db5db38613e73d2b668ec01bb239R72-R75, it identifies the max_prefill_length and sets prefill_buckets accordingly.